### PR TITLE
Fix raw Handlebars template compilation

### PR DESCRIPTION
**Pull request by kaukas**
Fixes issue #18.

### CODE REVIEW

File: test/test_ember_handlebars_template.rb [hunk 1]
```diff
@@ -145,6 +145,26 @@ def test_should_have_module_name_with_AMD_output
     end
   end
 
+  def test_compile_raw_template_and_strip_the_raw_suffix
+    with_ember_template 'Handlebars' do
+      asset = @env['app/templates/hi.raw.js']
```
Comment (tricknotes) [inline test/test_ember_handlebars_template.rb]: I think this file should be accessible without `.raw` extension.

Could you update [`Ember::Handlebars::Template.setup`](https://github.com/tricknotes/ember-handlebars-template/blob/v0.6.0/lib/ember/handlebars/template.rb#L23-L25) with `.raw` extension processor?
Such as the following:

``` ruby
env.register_engine '.raw.hbs', self, mime_type: 'application/javascript'
```

File: lib/ember/handlebars/helper.rb [hunk 2]
```diff
@@ -44,6 +44,10 @@ def template_path(path, config)
 
         path = path.split('/')
 
+        # Strip the '.raw' suffix if there is one.
```
Comment (tricknotes) [inline lib/ember/handlebars/helper.rb]: When https://github.com/tricknotes/ember-handlebars-template/pull/19#discussion_r52170894 is done, these lines are no longer necessary.

**Comment by tricknotes** [comment]
I left some comments.
However this PR is awesome :sunglasses:

**Comment by kaukas** [comment]
Makes sense. I updated the tests and extension registration.

**Comment by kaukas** [comment]
Travis tests look good. I forgot the "Do" in the last commit message "Do skip..." Please drop it. Other than that it should be good to go!

Thank you!

**Comment by kaukas** [comment]
Fixed the last commit:
- Dropped the "Do".
- Dropped the `.raw` extension stripping code.

**Comment by tricknotes** [comment]
Thanks for your great work, @kaukas !
I'll release a new version with this changes as soon as possible.

**Comment by kaukas** [comment]
Thank you for the merge and for the great work on this project!

**Comment by tricknotes** [comment]
Now I released [ember-handlebars-template@0.7.0](https://rubygems.org/gems/ember-handlebars-template/versions/0.7.0) :rocket:

**Pull request merged**